.idea/jenkinsSettings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
We should have .idea and other IDEA / IDE generated files added to our .gitignore, fwiw. It's not a huge deal, but just something we should clean up eventually.

Reply:
I didn't touch .gitignore file because i suppose that JIdea is managing that internally but on other hand i don't have that much experience with JIDEa IDE because previously i was working with Netbeans and Neatbeans was not generating lot of files , even you can open completely clean dir and he will generate all necessary files and then on exit will clean them up.
I have read doc. from jidea:  https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems And there is mentioned that just several files can be clean up and latest version of jidea is managing that in .gitignore file but still that's lot of generated content that needs to be shared.
BTW i have clean up left overs from VSCode and other not needed files so now should be fine , you can check it up.